platform/contact-center/training-video-process.md

```diff
@@ -1,9 +1,17 @@
-### Contact Center Training Video Process
+# Contact Center Training Video Process
 
-## Summary
-Training videos are the video version of the product guide and are created by VFS teams for contact center use.  The contact center training videos can easily be created by recording a Slack, Zoom, Microsoft Teams, or Skype call.  
+**Overview:**: How to create Product Videos for the Contact Center Review in the Build Stage of the Collaboration Cycle.
 
-During the video the team will walk though the primary flows of the product, using layman terms whenever possible. 
+**Audience**: VFS teams building on VA.gov
+
+## Process
+
+Training videos are the video version of the product guide and are created by VFS teams for contact centers to help Veterans troubleshoot VA.gov.  
+
+When your team is at the [Prepare for Contact Center Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md#prepare-for-contact-center-review) stage, reach out to the Contact Center POCs on #vsp-contact-center-support Slack channel to set up a video meeting. The contact center training videos will be created by recording the Zoom call so that we can capture the closed captioning.  
```

Comment:
If the team has actually filled out an issue for a contact center review, then I would recommend that we reach out to set up the demo meeting. Teams haven't been great about filling these out so maybe we leave as is for now and ask that they set up the meeting.

Reply:
Thank you for this level of detail Chante.  Because we have only done one demo video together, I was unaware of that Zoom ability.